host-ext/crouton/manifest.json

```diff
@@ -4,7 +4,7 @@
   "name": "crouton integration",
   "short_name": "crouton",
   "description": "Improves integration with crouton chroots.",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
This would be 2.1.1

Major: incompatible protocol changes or major new features
Minor: backwards-compatible protocol changes
Revision: minor stuff.